pkg/scheduler/plugins/usage/usage.go

```diff
@@ -0,0 +1,164 @@
+/*
+Copyright 2022 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package usage
+
+import (
+	"fmt"
+	"k8s.io/klog"
+	k8sFramework "k8s.io/kubernetes/pkg/scheduler/framework"
+	"strconv"
+	"strings"
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+)
+
+const (
+	// PluginName indicates name of volcano scheduler plugin.
+	PluginName        = "usage"
+	cpuUsageAvgPrefix = "CPUUsageAvg."
+	memUsageAvgPrefix = "MEMUsageAvg."
+	thresholdSection  = "thresholds"
+	cpuUsageAvg5m     = "5m"
+)
+
+/*
+   actions: "enqueue, allocate, backfill"
+   tiers:
+   - plugins:
+     - name: usage
+       arguments:
+          thresholds:
+            CPUUsageAvg.5m: 80
+            MEMUsageAvg.5m: 90
+*/
+
+type thresholdConfig struct {
+	cpuUsageAvg map[string]float64
+	memUsageAvg map[string]float64
+}
+
+type usagePlugin struct {
+	pluginArguments framework.Arguments
+	weight          int
+	threshold       thresholdConfig
+}
+
+// New function returns usagePlugin object
+func New(args framework.Arguments) framework.Plugin {
+	usageWeight := 1
```

Comment:
What does `usageWeight` mean? Where is the validation?

Reply:
it's the plugin weight. like binpack.weight

Reply:
Got it! That's reasonable. Sussget adding some comment to make it clear.